src/components/OSCALMarkupProse.js

```diff
@@ -1,32 +1,35 @@
 import React from "react";
 import ReactMarkdown from "react-markdown";
 
+import OpenInNewIcon from "@mui/icons-material/OpenInNew";
+
+const components = {
+  a: ({ ...props }) =>
+    props.href.startsWith("http") ? (
+      <a target="_blank" {...props}>
+        <OpenInNewIcon /> {props.children}
+      </a>
+    ) : (
+      <a {...props}>{props.children}</a>
+    ),
+};
+
 /**
  * Renders a string of markdown to React elements
  * @param {String} props.text
  * @returns a React element from the markdown
  */
 
 export function OSCALMarkupMultiLine(props) {
+  components.p = props.paragraphComponent ?? "p";
```

Comment:
This changes the base `components` object in a way that I am not especially comfortable with.

```tsx
<ReactMarkdown
  components = {{
    ...components,
    p: props.paragraphComponent ?? "p",
  }}
>
  {props.children}
</ReactMarkdown> 
```